EIPS/eip-acceptance.md

```diff
@@ -0,0 +1,29 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: EIP Community Veto
+    Author: Sam Griffin (sam.griffin@gmail.com)
+    Type: Meta
+    Status: Draft
+    Created: 2018-02-24
+    Modifies: EIP-1
+
+## Simple Summary
+The greater Ethereum community should be able to veto EIPs they don't like. Also, the process by which EIPs become part of the Ethereum standard should be clarified. 
+
+## Abstract
+Some EIPs have contentious moral/ethical/legal/philosophical rammifications. The process of drafting the EIP where the author comes up with what they think is the best form of their idea should be free from interference, but the community should have a voice to prevent unpopular EIPs becoming part of the standard. 
+
+## Motivation
+The community currently has no official procedure for indicating their dislike of certain EIPs. This leads to brigading of GitHub pull requests that make it more difficult for EIP editors and draft writers. Additionally, EIP draft writers do not have specific data on how controversial their EIP is. Controversial hard forks should be avoided whenever possible, and if compromise can happen in the EIP drafting stage, this could avoid community splits. The current role of the AllCoreDevs group is also not detailed in EIP-1, and this should be made explicit.
+
+## Specification
+I propose two new phases to the EIP process: "Staging" and "Implementing"
+Once an EIP champion has polished their draft to their satisfaction, it will go to the "Staging" phase. At this point a carbon vote shall be set up and the draft shall be sent to the AllCoreDevs group. The community shall be given a period of a week to register their carbon votes. If the majority of the carbon vote is against the EIP in its current form, it shall be sent back to the "Draft" phase.
```

Comment:
> Once an EIP champion has polished their draft to their satisfaction, it will go to the "Staging" phase.

This is exactly what merging an EIP as a DRAFT is, what are the advantages to renaming it to STAGING?  What is unclear about the DRAFT name?

Reply:
My idea is that at the Draft phase, it is undergoing revision, whereas the "Staging" phase would be for evaluating EIPs that are in a stable state.

It's hard to evaluate things as they're being changed.

Reply:
The staging phase would have a community vote window, if that's what you mean by delivery date. Client software probably shouldn't be implementing any EIPs until AllCoreDevs and the community carbon votes have passed.

It's helpful to have the EIP frozen when the community is voting on it, though, so that it's clear that everyone is voting an exactly the same thing.

Reply:
I meant "promotion" of the standard, not "delivery"